docs/manual-CN/2.query-language/4.statement-syntax/1.data-definition-statements/create-tag-edge-syntax.md

```diff
@@ -80,10 +80,10 @@ nebula> CREATE TAG player_with_default(name string, age int DEFAULT 20)  -- 默
 nebula> CREATE EDGE follow_with_default(start_time timestamp DEFAULT 0, grade double DEFAULT 0.0)  -- 默认 start_time 设置为 0，默认 grade 设置为 0.0
 ```
 
-<!-- ```ngql
+```ngql
 nebula> CREATE TAG woman(name string, age int,
    married bool, salary double, create_time timestamp)
-   TTL_DURATION = 100, TTL_COL = create_time -- 过期时间是 100， 从当前时间开始
+   TTL_DURATION = 100, TTL_COL = create_time -- 时间间隔是 100s，从 create_time 字段的值开始
```

Comment:
To support the drop ttl attribute, column names after ttl_col need to be quoted
TTL_COL = “create_time“